### PR TITLE
Migrate fps camera

### LINKED ISSUE #217: Migrate FPS view controller
Migrate the `FPS (first-person)` view controller to RViz for ROS2.

### PR DESCRIPTION
**Pull request by Martin-Idel-SI**
Closes #217 

* Linux [Build Status](http://ci.ros2.org/job/ci_linux/4556/)
* Linux-aarch64 [Build Status](http://ci.ros2.org/job/ci_linux-aarch64/1466/)
* macOS [Build Status](http://ci.ros2.org/job/ci_osx/3741/)
* Windows [Build Status](http://ci.ros2.org/job/ci_windows/4636/)

### CODE REVIEW

**Comment by Martin-Idel-SI** [comment]
CI after rebase:
* Linux [Build Status](http://ci.ros2.org/job/ci_linux/4600/)
* Linux-aarch64 [Build Status](http://ci.ros2.org/job/ci_linux-aarch64/1502/)
* macOS [Build Status](http://ci.ros2.org/job/ci_osx/3781/)
* Windows [Build Status](http://ci.ros2.org/job/ci_windows/4678/)

**Comment by Martin-Idel-SI** [comment]
CI after rebase:
* Linux [Build Status](http://ci.ros2.org/job/ci_linux/4615/)
* Linux-aarch64 [Build Status](http://ci.ros2.org/job/ci_linux-aarch64/1516/)
* macOS [Build Status](http://ci.ros2.org/job/ci_osx/3799/)
* Windows [Build Status](http://ci.ros2.org/job/ci_windows/4693/)

File: rviz_default_plugins/CMakeLists.txt [hunk 1]
```diff
@@ -428,7 +429,20 @@ if(BUILD_TESTING)
...
+      test/rviz_default_plugins/scene_graph_introspection.cpp
+      APPEND_ENV AMENT_PREFIX_PATH=${CMAKE_INSTALL_PREFIX} PATH=${CMAKE_INSTALL_PREFIX}/bin;${CMAKE_INSTALL_PREFIX}/opt/rviz_assimp_vendor/bin;${CMAKE_INSTALL_PREFIX}/opt/rviz_yaml_cpp_vendor/bin;${CMAKE_INSTALL_PREFIX}/opt/rviz_ogre_vendor/bin)
+    if(TARGET fps_view_controller_test)
+      target_include_directories(fps_view_controller_test PUBLIC
+        ${TEST_INCLUDE_DIRS})
+      target_link_libraries(fps_view_controller_test
+        ${TEST_LINK_LIBRARIES})
+    endif()
+
+if(EnableVisualTests STREQUAL "True")
```
Comment (wjwwood) [inline rviz_default_plugins/CMakeLists.txt:445]: nitpick: indentation looks wrong here. Could be fixed in a small follow up pr.

**Review by wjwwood — APPROVED** [review]

File: rviz_default_plugins/CMakeLists.txt (hunk 1 above)
Comment (Martin-Idel-SI) [reply rviz_default_plugins/CMakeLists.txt:445]: This will be done on the open PR #289

**Pull request merged**